boards/arm/nucleo_u5a5zj_q/doc/index.rst

```diff
@@ -0,0 +1,377 @@
+.. _nucleo_u5a5zj_q_board:
+
+ST Nucleo U5A5ZJ Q
+##################
+
+Overview
+********
+
+The Nucleo U5A5ZJ Q board, featuring an ARM Cortex-M33 based STM32U5A5ZJ MCU,
+provides an affordable and flexible way for users to try out new concepts and
+build prototypes by choosing from the various combinations of performance and
+power consumption features. Here are some highlights of the Nucleo U5A5ZJ Q
+board:
+
+
+- STM32U5A5ZJ microcontroller in LQFP144 package
+- Internal SMPS to generate V core logic supply
+- Two types of extension resources:
+
+  - Arduino Uno V3 connectivity
+  - ST morpho extension pin headers for full access to all STM32 I/Os
+
+- On-board ST-LINK/V3E debugger/programmer
+- Flexible board power supply:
+
+   - USB VBUS or external source(3.3V, 5V, 7 - 12V)
+   - ST-Link V3E
+
+- Three users LEDs
+- Two push-buttons: USER and RESET
+- USB Type-C |trade| Sink device FS
+
+Hardware
```

Comment:
you may want to add a picture of the board?

Reply:
Unfortunately I cannot find an image of the board on ST website. The Nucleo U5A5 page displays a Nucleo U575.